src/vm/win32threadpool.cpp

```diff
@@ -392,11 +393,11 @@ BOOL ThreadpoolMgr::Initialize()
 
     // initialize Worker and CP thread settings
     DWORD forceMin;
-    forceMin = CLRConfig::GetConfigValue(CLRConfig::INTERNAL_ThreadPool_ForceMinWorkerThreads);
+    forceMin = Configuration::GetKnobDWORDValue(W("System.Threading.ThreadPool.MinThreads"), CLRConfig::INTERNAL_ThreadPool_ForceMinWorkerThreads);
```

Comment:
Sure, done. I added helper functions for getting min/max. For now, the behavior will be the same though (each check results in a lookup). I think we should add a caching mechanism like we have for the old CLRConfig types, but that can be a separate change (not rushed for RC2).